inst/pages/Introductory-workflow/Introductory.qmd

```diff
@@ -0,0 +1,293 @@
+# Introductory
+
+<div id="workflow">
+
+<img src="images/logo.svg" alt="logo workflow" id="banner"/>
+
+# Introduction
+
+Hello and welcome to a complete workflow using the latest R/Bioconductor tools for microbiome data science. In this tutorial we’ll walk you through some basic steps of a composition analysis study using OMA. These will be applicable to almost any of your projects and will help you understand the fundamental concepts that will sky-rocket 🚀 your future microbiome analyses.
+
+# Importing data
+
+When using microbiome packages, there are many different ways to import your data. Let's first load the mia package:
+
+```{r}
+#| output: false
+library(mia)
+```
+
+```{r}
+#| echo: false 
+#| output: false
+library(DT)
+```
+
+You can either choose to use your own data or one of the built-in datasets provided by mia which you may find in the [**OMA book**](https://microbiome.github.io/OMA/docs/devel/pages/04_containers.html#sec-example-data) :
+
+In this tutorial, we'll be using the [Tengeler2020](https://microbiome.github.io/mia/reference/Tengeler2020.html) dataset. This dataset was made by A.C. Tengeler to try to demonstrate the impact of altered microbiomes on the brain structure. Here's how we can load the data into our R environment :
+
+```{r}
+data("Tengeler2020", package="mia")
+tse <- Tengeler2020
+```
+
+There are of course other ways to import your data using the [**mia package**](https://microbiome.github.io/mia/). These include: using either [**your own data**](https://microbiome.github.io/OMA/docs/devel/pages/04_containers.html#sec-import-from-file) or by converting an existing object to a *TreeSummarizedExperiment* object as stated in [**this section of the OMA book**](https://microbiome.github.io/OMA/docs/devel/pages/04_containers.html#conversions-between-data-formats-in-r).
+
+<hr>
+
+# Storing microbial data
```

Comment:
This section could be shortened (remove subsections on assays, colData, rowData), see if something could be moved to OMA chapter that deals with these and then here just provide very brief examples and refer the reader there.

Reply:
I shortened this section. However wouldn't it be beneficial to keep some longer explanation in the French and Dutch version of the workflow ?

Reply:
Ok